src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -110,10 +107,27 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
                         new AcquireTokenCommonParameters(), 
                         requestContext));
 
-                accounts = await cacheSessionManager.GetAccountsAsync(Authority).ConfigureAwait(false);
+                localAccounts = await cacheSessionManager.GetAccountsAsync(Authority).ConfigureAwait(false);
+            }
+
+            if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
```

Comment:
I don't think the CanBrokerSupportSilentAuth flag is needed for GetAccounts and RemoveAccounts. iOS broker can simply do nothing...

Reply:
This will be a cleaner design. 

You can probably eliminate it from AcquireTokenSilent as well, but it will be more complicated.